### PR TITLE
Feature/bex 1171/error redirect

### PR DESCRIPTION
**Pull request by ekkinox**
See https://oat-sa.atlassian.net/browse/NEX-1171

### CODE REVIEW

File: Tests/Functional/Action/Tool/Message/MessageActionTest.php [hunk 1]
```diff
@@ -181,6 +183,68 @@ public function testItReturnsUnauthorizedResponseWithEmptyIdToken(): void
...
+
+        $response = $this->client->getResponse();
+        $this->assertInstanceOf(RedirectResponse::class, $response);
+        $this->assertEquals(
+            'http://platform.com/error?lti_errormsg=LTI+message+request+authentication+failed%3A+JWT+id_token+is+expired',
+            $response->getTargetUrl()
+        );
+    }
+
+    public function testItRedirectOnErrorToLaunchPresentationReturnUrlWithQueryParamsWhenSpecifiedInMessageClaims(): void
```
Comment (m-lagarde) [inline Tests/Functional/Action/Tool/Message/MessageActionTest.php]: we can do a bit longer: it's `testItRedirects` :smiley:

Comment (ekkinox) [reply Tests/Functional/Action/Tool/Message/MessageActionTest.php]: ok, fixed

**Review by m-lagarde — APPROVED** [review]
Works fine, good job!
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Code is self documented

**Review by emgolubev — APPROVED** [review]
It works for me.

**Comment by emgolubev** [comment]
- [x] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [x] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Code is self-documented

**Pull request merged**